README.md

```diff
@@ -5,12 +5,6 @@
 
 **Umbrella Project**: [Chef Foundation](https://github.com/chef/chef-oss-practices/blob/master/projects/chef-foundation.md)
```

Comment:
I think we should remove all project metadata - this is now handled by the primaryApplication custom property.